akka-serialization-jackson/src/test/scala/akka/serialization/jackson/JacksonSerializerSpec.scala

```diff
@@ -238,7 +238,7 @@ class JacksonJsonSerializerSpec extends JacksonSerializerSpec("jackson-json") {
             serializerIdentifier: Int,
             configuredModules: immutable.Seq[Module]): immutable.Seq[Module] =
           if (serializerIdentifier == JacksonJsonSerializer.Identifier) {
-            configuredModules.filterNot(_.isInstanceOf[AfterburnerModule])
+            configuredModules.filterNot(_.isInstanceOf[JavaTimeModule])
```

Comment:
I think the assertions below don't exercise the customised settings so I don't see how we are asserting the customization works. The code in `withSystem` (below, lines 251-262) could try to serialise a `java.time.Instant` and we'd expect an exception (or whatever jackson produces in that case) because `JavaTimeModule` is disabled.

Reply:
ok, improved it in 80a479b